README.md

```diff
@@ -13,37 +13,58 @@ In March 2015 we've also written a short paper outlining discussing some develop
 
 **Note:** You can also find a detailed usage description [here](docs/basic-usage.md).
 
-To use the SQL parser in your own projects you simply have to follow these few steps.
+
+### Build and Test
 
  1. Download the [latest release here](https://github.com/hyrise/sql-parser/releases)
  2. Compile the library `make` to create `libsqlparser.so`
  3. *(Optional, Recommended)* Run `make install` to copy the library to `/usr/local/lib/`
```

Comment:
Not yours, but I wouldn't consider it "recommended" to install the library. I would be rather surprised if somebody installs it system-wide.